controllers/defaults/placements.go

```diff
@@ -128,16 +131,17 @@ var (
 
 // getTopologySpreadConstraintsSpec populates values required for topology spread constraints.
 // TopologyKey gets updated in newStorageClassDeviceSets after determining it from determineFailureDomain.
-func getTopologySpreadConstraintsSpec(maxSkew int32) corev1.TopologySpreadConstraint {
+func getTopologySpreadConstraintsSpec(maxSkew int32, label string) corev1.TopologySpreadConstraint {
 	topologySpreadConstraints := corev1.TopologySpreadConstraint{
 		MaxSkew:           maxSkew,
 		TopologyKey:       corev1.LabelHostname,
 		WhenUnsatisfiable: "ScheduleAnyway",
 		LabelSelector: &metav1.LabelSelector{
 			MatchExpressions: []metav1.LabelSelectorRequirement{
 				{
-					Key:      osdPvcLabelSelector,
-					Operator: metav1.LabelSelectorOpExists,
```

Comment:
If we go with the above suggestion, need to use this operator in case of the prepare job